content/collections/workflow/en/experiment-learnings.md

```diff
@@ -13,59 +13,113 @@ You’ve designed your experiment, rolled it out to your users, and given them e
 {{/partial:admonition}}
 
-Amplitude will not generate p-values or confidence intervals for experiments using binary metrics (i.e., unique conversions) until each variant has 100 users **and** 25 conversions. Experiments using non-binary metrics need only to reach 100 users per variant.
+The Summary card displays a badge labeled *Significant* if the experiment reached statistical significance, and a badge labeled *Not Significant* if it didn't. This card can display several badges at once:
 
-## View results
+* *Inconclusive*: the test was inconclusive for the primary metric.
+* *Above Goal* or *Below Goal:* the primary metric's mean was either **above** or **below** its goal depending on the direction of the test (increase = above, decrease = below).
+* *Above Control* or *Below Control:* the primary metric's mean was either **above** or **below** the control's mean, depending on the direction of the test (increase = above, decrease = below). These badges are only relevant to stat sig results.
 
-To generate and view experimental results, follow these steps:  
+![summary.png](/docs/output/img/workflow/summary-png.png)
 
-1. In your experiment, the *Activity* page includes two sections to view your results. The *Summary* section and the *Analysis* card. The *Summary* section will describe your experiment's hypothesis and note whether it has or has not reached statistical significance.  
 
-    An experiment is said to be **statistically significant** when we can confidently say that the results are highly unlikely to have occurred due to random chance. (More technically, it’s when we reject the null hypothesis.) That might sound pretty subjective, but it’s grounded solidly in statistics. Stat sig relies on a variant’s **p-value**, which is the probability of observing the data we see, assuming there is no difference between the variant and the control. If this probability drops below a certain threshold (statisticians refer to this threshold as the **alpha**), then we consider our experiment to have achieved statistical significance.
+## Analysis card
 
-    The *Summary* section will display a badge labeled *Significant* if stat sig was met, and a badge labeled *Not Significant* if stat sig was not met.
+At the top of the Analysis card is an overview that explains how your experiment performed, broken down by metric and variant. Below that, a collection of experiment results charts, which you can analyze by metric, display information about:
 
-    The *Summary* section may include multiple badges simultaneously:
+* Confidence intervals
+* Cumulative exposure*
```

Comment:
nit: extra "*" at end of line

Reply:
That signifies it's related to the information below `* Cumulative exposure tracks the experiment as a whole.`

<img width="812" alt="image" src="https://github.com/user-attachments/assets/f7cddbfa-50c3-48e9-be00-2166ee7eca42" />

Reply:
Nm, removed this as part of the last comment.